app/views/shared/_footer.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="container">
+  <footer class="d-flex flex-wrap justify-content-between align-items-center py-3 my-4 border-top">
+    <div class="col-md-4 d-flex align-items-center">
+      <a href="/" class="mb-3 me-2 mb-md-0 text-muted text-decoration-none lh-1">
+        <svg class="bi" width="30" height="24"><use xlink:href="#bootstrap"></use></svg>
+      </a>
+      <span class="text-muted">© 2022 SiTripio ~ Made with ❤️ ~ </span>
```

Comment:
Changed the red heart for the back and white used by Yauheni. This will update in the next pull